workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/components/Table/index.css

```
@@ -16,6 +16,10 @@
 	max-width: 110px;
 }
 
+.table-layout-auto {
+	table-layout: auto !important;
```

```suggestion
	table-layout: auto;
```
Is that !important statement required?